development_workflow.md

```diff
@@ -0,0 +1,69 @@
+# Development / Contribution Guidelines
+
+## General Development Workflow in the Lab
+
+
+
+### Forking and Branching
```

Comment:
We should link to our [introduction to GitHub/git guide](https://github.com/SeattleTestbed/docs/blob/master/Archive/Local/RepoAccess.md) for newcomer to the lab.  According to Albert's [comment in another issue](https://github.com/secure-systems-lab/code-style-guidelines/issues/6#issuecomment-285608378), the images are missing but have been gitified.